src/common/helpers/errors.js

```diff
@@ -1,21 +1,21 @@
-import { statusCodes } from '../constants/status-codes.js'
+import { constants as httpConstants } from 'http2'
```

Comment:
I would use `node:http2` to reduce the attack surface by negating spoofing of inbuilt npm packages.